### PR TITLE
[stable/heapster] Add annotations to service

### PR DESCRIPTION
**Pull request by wwojcik**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/heapster/values.yaml [hunk 1]
```diff
@@ -40,6 +40,12 @@ service:
   #   kubernetes.io/name: "Heapster"
   labels:
 
+  ## Here annotations can be added to the heapster service
+  # annotations:
+  #   prometheus.io/path: /metrics
+  #   prometheus.io/port: "8082"
+  #   prometheus.io/scrape: "true"
+  annotations: {}
```
Comment (rjkernick) [inline stable/heapster/values.yaml:48]: Please add empty default to the readme

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/heapster/values.yaml (hunk 1 above)
Comment (wwojcik) [reply stable/heapster/values.yaml:48]: Thanks, added to README

**Pull request merged**